ubid.rb

```diff
@@ -69,6 +69,7 @@ class UBID
   TYPE_FIREWALL = "fw"
   TYPE_POSTGRES_FIREWALL_RULE = "pf"
   TYPE_GITHUB_REPOSITORY = "gp"
+  TYPE_LOAD_BALANCER = "1b"
```

Comment:
this is `1b`, why not `lb`?

Reply:
I had it `lb` first, but then the tests failed. I searched some and found out that the algorithm we use for UBID calculation maps the chars 1, l, i, all to 1. That's why we must use only 1 for all of those. https://github.com/ubicloud/ubicloud/blob/4ea8e725561fd35e74032a3f469e7ab7b97df9a6/spec/ubid_spec.rb#L133

Reply:
```
    # In crockford's base32 multiple characters can map to a single number,
    # for example ["1","I","L"] all map to 1. However, the reverse (number
    # -> character) only uses one canonical one (e.g. 1 maps to "1").
    # This test ensures we only use canonical characters in our type
    # constants, so ids are actually prefixed by those constants.
    #
    # See https://www.crockford.com/base32.html
```